distribution/lib/Standard/Table/0.0.0-dev/src/Data/Column_Ref.enso

```diff
@@ -0,0 +1,9 @@
+from Standard.Base import all
+
+## Reference to a column in a table.
+type Column_Ref
+    ## Reference to a column by name in a table.
+    Name name:Text
+
+    ## Reference to a column by index in a table.
+    Index index:Integer
```

Comment:
I moved `Column_Ref` to a separate file, because I think it may be used in other contexts as well so it seemed sensible to have it in its own file.

Reply:
:+1: